src/main/java/org/opentripplanner/updater/vehicle_rental/VehicleRentalUpdater.java

```diff
@@ -164,14 +167,13 @@ private void applyStations(Graph graph) {
                 VehicleRentalStationVertex vertex = verticesByStation.get(station);
                 if (vertex == null) {
                     makeVertex(graph, station);
-                } else if (station.x != vertex.getX() || station.y != vertex.getY()) {
+                } else if (Math.abs(station.x - vertex.getX()) > 0.00005 || Math.abs(station.y - vertex.getY()) > 0.00005) {
```

Comment:
Move to constant, maybe shared with car updater

Reply:
Will do